src/worker.js

```diff
@@ -25,6 +26,13 @@ worker.webHookHandler = async function(message, context) {
       ref: message.payload.details['event.base.repo.branch'],
     });
 
+    // Check if this is meant to be built by tc-github at all.
+    // This is a bit of a hack, but is needed for bug 1274077 for now
+    let c = yaml.safeLoad(taskclusterConfig);
+    if (c.tasks && c.tasks.length > 0 && _.has(c.tasks[0], 'extra.github')) {
```

Comment:
So perhaps this should be a filter that kicks out tasks that don't have `extra.github` instead?